plugins/occt/CMakeLists.txt

```diff
@@ -27,8 +33,8 @@ option(F3D_PLUGIN_OCCT_COLORING_SUPPORT "Enable coloring support in occt plugin"
 mark_as_advanced(F3D_PLUGIN_OCCT_COLORING_SUPPORT)
 
 if(F3D_PLUGIN_OCCT_COLORING_SUPPORT)
-  if (NOT (TARGET "TKXDESTEP") OR NOT (TARGET "TKXDEIGES"))
-      message(FATAL_ERROR "occt plugin: TKXDESTEP and TKXDEIGES OCCT modules are not found. Turn off F3D_PLUGIN_OCCT_COLORING_SUPPORT or enable them in your OpenCascade build.")
+  if (NOT (TARGET "TKXSDRAWSTEP") OR NOT (TARGET "TKXSDRAWIGES"))
```

Comment:
the name of the modules changes, you need to check the version first before checking. I'd suggest putting the names in a CMake list.